src/models.py

```diff
@@ -191,7 +194,9 @@ class Metadata(MetadataPayloadData):
     name: str
     license: LicenseEnum
     platform: PlatformEnum
-    aquisition_date: datetime
+    year: int
+    month: int
+    day: int
```

Comment:
sorry,
I just created the columns as `aquisition_year` `aquisition_month` and `aquisition_day` to be more precise here